### PR TITLE
Update OBR Segment IG Documentation

### PR DESCRIPTION
**Pull request by jorg3lopez**
Updated OBR segment IG documentation for required and currently captured fields

Describe what changed in this PR at a high level.

#493 

## Checklist

- [n/a] I have added tests to cover my changes
- [n/a] I have added logging where useful (with appropriate log level)
- [n/a] I have added JavaDocs where required
- [n/a] I have updated the documentation accordingly

### CODE REVIEW

File: ig/input/fsh/nbs-obr-segment.fsh [hunk 1]
```diff
@@ -2,28 +2,31 @@ Logical: OBRSegment
...
+* universalServiceIdentifierIdentifier 1..1 SU string "OBR-4.1 Universal Service Identifier Identifier"
+* universalServiceIdentifierText 1..1 SU string "OBR-4.2 Universal Service Identifier Text"
+* universalServiceIdentifierNameOfCodingSystem 1..1 SU string "OBR-4.3 Universal Service Identifier Name Of Coding System"
+* universalServiceIdentifierOriginalText 1..1 SU string "OBR-4.9 Universal Service Identifier Original Text"
+* observationDateTime 1..1 SU dateTime "OBR-7 Observation Date/Time"
 // other optional fields
 
 
 // Provenance Section begins
 Instance: segment-obr-logical-model-history-create
```
Comment (sfradkin) [inline ig/input/fsh/nbs-obr-segment.fsh:21]: Please keep the original Provenance instance definition and create a new one for the new update.  Each Provenance instance becomes a line on the history tab for the OBR segment model.

File: ig/input/fsh/nbs-obr-segment.fsh [hunk 2]
```diff
@@ -27,3 +28,17 @@ Usage: #definition
...
+Instance: segment-obr-logical-model-history-create
+InstanceOf: Provenance
+Title: "Removal of OBR-3, OBR-8, OBR-15 and OBR-16 segments"
+Usage: #definition
+* target[+] = Reference(StructureDefinition/segment-obr-logical-model)
+* recorded = "2023-09-07T00:00:00.0000Z"
+* occurredDateTime = "2023-09-07"
+* reason = http://terminology.hl7.org/CodeSystem/v3-ActReason#METAMGT
+* reason.text = "Updated segments to match the ones we initially support"
+* activity = http://terminology.hl7.org/CodeSystem/v3-DataOperation#CREATE
```
Comment (sfradkin) [inline ig/input/fsh/nbs-obr-segment.fsh:42]: the `activity` should be `UPDATE`.  double check at the terminology URL for the `v3-DataOperation` code set

**Review by sfradkin — CHANGES_REQUESTED** [review]
couple more issues

**Review by halprin — APPROVED** [review]
Other than @sfradkin's comments on provenance, I like!  Nice work!

File: ig/input/fsh/nbs-obr-segment.fsh [hunk 3]
```diff
@@ -2,21 +2,22 @@ Logical: OBRSegment
...
+* universalServiceIdentifierIdentifier 1..1 SU string "OBR-4.1 Universal Service Identifier Identifier"
+* universalServiceIdentifierText 1..1 SU string "OBR-4.2 Universal Service Identifier Text"
+* universalServiceIdentifierNameOfCodingSystem 1..1 SU string "OBR-4.3 Universal Service Identifier Name Of Coding System"
+* universalServiceIdentifierOriginalText 1..1 SU string "OBR-4.9 Universal Service Identifier Original Text"
+* observationDateTime 1..1 SU dateTime "OBR-7 Observation Date/Time"
 // other optional fields
 
 
 // Provenance Section begins
-Instance: segment-obr-logical-model-history-create
```
[the remark this answers is not in the document]
Comment (jorg3lopez) [reply ig/input/fsh/nbs-obr-segment.fsh]: Good catch. Done.

File: ig/input/fsh/nbs-obr-segment.fsh (hunk 2 above)
Comment (jorg3lopez) [reply ig/input/fsh/nbs-obr-segment.fsh]: done.

File: ig/input/fsh/nbs-obr-segment.fsh [hunk 4]
```diff
@@ -27,3 +28,17 @@ Usage: #definition
 * activity = http://terminology.hl7.org/CodeSystem/v3-DataOperation#CREATE
 * agent[+].type = http://terminology.hl7.org/CodeSystem/provenance-participant-type#author
 * agent[=].who.display = "T. R. Johnson"
+
+
+Instance: segment-obr-logical-model-history-create
```
[the remark this answers is not in the document]
Comment (jorg3lopez) [reply ig/input/fsh/nbs-obr-segment.fsh]: done.

**Review by sfradkin — APPROVED** [review]
changes look good

**Pull request merged**